templates/common/copilot-plugin-existing-api-api-key/teamsapp.yml.tpl

```diff
@@ -17,6 +17,15 @@ provision:
     writeToEnvironmentFile:
       teamsAppId: TEAMS_APP_ID
 
+  # Register API KEY
+  - uses: apiKey/create
+    with:
+      name: {{ApiSpecAuthName}}
+      appId: ${{TEAMS_APP_ID}}
+      apiSpecPath: {{{ApiSpecPath}}}
+    writeToEnvironmentFile:
+      registrationId: {{ApiSpecAuthRegistrationIdPrefix}}_REGISTRATION_ID
```

Comment:
We may support multiple api keys in the future, so better to use name instead of prefix here.

Reply:
Thanks. updated